### PR TITLE
feature/BI improvement FM

### PR DESCRIPTION
**Pull request by vinulw**
### feature/BI improvement - fmp
A new `fm` level `SiteItem` applies `BIWaitingPeriod` and `BIPOI` (period of indemnity). 
- This new level is applied before the `SiteCoverage` level. 
- Output loss from this level is calculated as `output_loss = min(input_loss - BIWaitingPeriod, BIPOI)`.
- For multi peril the terms are in accordance with the `alloc_rule_il`, by default allocation will occur proportional to the input loss.

### CODE REVIEW

File: oasislmf/utils/defaults.py [hunk 1]
```diff
@@ -225,7 +225,9 @@ def get_default_unified_profile(path=False):
 
 def get_default_fm_aggregation_profile(path=False):
     fp = os.path.join(STATIC_DATA_FP, 'default_fm_agg_profile.json')
-    return {int(k): v for k, v in get_default_json(src_fp=fp).items()} if not path else fp
```
Comment (sstruzik) [inline oasislmf/utils/defaults.py:228]: I don't understand the point of path in the original function and looking at the code path is never used as a parameter.
I think we can clean this and remove it. this way we have get_default_fm_aggregation_profile that return the default profile.

Comment (vinulw) [reply oasislmf/utils/defaults.py:228]: the path variable seems to be redundant in these functions as well: 
- `get_default_accounts_profile`
- `get_default_exposure_profile`
- `get_default_fm_profile_field_values`
- `get_default_step_policies_profile`
- `get_config_profile`
- `get_default_unified_profile`
